code/modules/vore/fluffstuff/custom_clothes_vr.dm

```diff
@@ -1003,6 +1003,15 @@
 	body_parts_covered = UPPER_TORSO|LOWER_TORSO|ARMS
 	allowed = list (/obj/item/weapon/material/knife)
 
+/obj/item/clothing/under/taiga
```

Comment:
Should be something like
`/obj/item/clothing/under/fluff/taiga`
Since fluff items have fluff infront of them.